api/drf_views.py

```diff
@@ -255,7 +256,11 @@ def get_serializer_class(self):
         if self.action == 'mini_events':
             return ListMiniEventSerializer
         elif self.action == 'list':
-            return ListEventSerializer
+            request_format_type = self.request.GET.get('format', 'json')
```

Comment:
```suggestion
            request_format_type = self.request.GET.get('format', 'json')
            is_tableau = self.request.GET.get('tableau', 'false').lower()
            if request_format_type == 'csv' or is_tableau == 'true':
```

FYI - Couldn't figure it out yet how to work with this suggestion thingy. Consider :260 as changed to the above as well (last line). This is how I'll work with the Tableau WDC so I only need the CSV Serializer but with the JSON format. If you'll have the final version of how we're going to include the "name fields" into the generalized `CsvField()` please let me know.

Reply:
@kamicut can you clarify on CsvField? I think for now we've just decided to go with more verbose Method fields and adding model methods? cc @GregoryHorvath

Reply:
@batpad @kamicut No worries I already merged and started doing it the new way, but any way it would be easy for me to change my current code if we'd change it again.